runng/Cargo.toml

```diff
@@ -20,7 +20,7 @@ codecov = { repository = "jeikabu/runng", branch = "master", service = "github"
 pipes = []
 
 [dependencies]
-runng-sys = { version = "1.1.1", path = "../runng-sys" }
+runng-sys = { version = "1.1.1-rc", path = "../runng-sys" }
```

Comment:
Can't you just drop the version string?

Reply:
It has to have it when/if you `cargo publish`

Reply:
Oh... nvm then